debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/SnapshotProcessor.java

```diff
@@ -209,12 +212,12 @@ private void processResultSet(TableMetadata tableMetadata, ResultSet resultSet)
         while (rowIter.hasNext()) {
             if (isRunning()) {
                 Row row = rowIter.next();
-                WriteTimeHolder writeTimeHolder = new WriteTimeHolder();
-                RowData after = extractRowData(row, tableMetadata.getColumns(), partitionKeyNames, clusteringKeyNames, writeTimeHolder);
+                Object executionTime = readExecutionTime(row);
```

Comment:
Seems this could be `long` instead of `Object` as per the cast further below?

Reply:
Hm, it seems that executionTime can possibly be null, thus we don't want to cast it into long before verifying that it's not null..?
https://github.com/debezium/debezium-incubator/blob/master/debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/SnapshotProcessor.java#L250

Reply:
> Seems this could be `long` instead of `Object` as per the cast further below?

Hm, it seems that executionTime can possibly be null, thus we don't want to cast it into long before verifying that it's not null..?
https://github.com/debezium/debezium-incubator/blob/master/debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/SnapshotProcessor.java#L250